spec/requests/self_assessment_pages_spec.rb

```diff
@@ -53,7 +54,8 @@
       it "lets AC edit their self assessment" do
         page.should have_selector('textarea#self_assessment_response',
                                     text: 'These are some notes that I have written')
-        fill_in 'Response', with: 'Now I have edited my self-assessment.'
+        fill_in 'Performance Assessment', with: 'Now I have edited my self-assessment.'
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.